src/util/fs_utils.js

```diff
@@ -33,17 +31,12 @@ function file_must_not_exist(file_path) {
 
 
 /**
- *
  * file_must_exist
- *
  */
-function file_must_exist(file_path) {
-    return fs.statAsync(file_path).then(() => {
-        // do nothing. 
-    });
+async function file_must_exist(file_path) {
+    await fs.promises.stat(file_path);
```

Comment:
do we need to add a return? 
in the original function, we return empty.

Reply:
no need. that was done just because we had to return a promise in order to wait for it.
now with await we don't need to, and in this case the function always returned undefined.
thanks, resolving.